api/app/settings/common.py

```diff
@@ -978,6 +978,15 @@
     "FLAGSMITH_ON_FLAGSMITH_SERVER_API_URL", default=FLAGSMITH_ON_FLAGSMITH_API_URL
 )
 
+FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_ENVIRONMENT_ID = env.int(
+    "FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_ENVIRONMENT_ID",
+    default=None,
+)
+FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_TAG_ID = env.int(
```

Comment:
For future proofing, should this be `env.list` ?

Reply:
I really didn't think so given the requirements. From a UI perspective, it made more sense to me that people tag all the features that are supposed to be exported in one tag, rather than trying to combine tags together which seemed more error prone. For example, if we're using existing tags for the feature export and someone unaware of the feature export process adds a new tag to an existing one it could leak that feature by accident.

Reply:
That being said, I can totally update this if you disagree. Ultimately I'm not the user here, it's the whole Flagsmith team.